src/content-scripts/linkExposure.content.js

```diff
@@ -3,29 +3,43 @@
  * @module webScience.linkExposure.content
  */
 
-import { importMatchPatternSet } from "../matching.js";
-import { parseFacebookLinkShim, parseAmpUrl } from "../linkResolution.js";
+import { urlToPS1 } from "../linkResolution.js";
 
-// async IIFE wrapper to enable await syntax
+// async IIFE wrapper to enable await syntax and early returns
 (async function () {
 
+    // If the linkExposure content script is already running on this page, no need for this instance
+    if("webScience" in window) {
+        if("linkExposureActive" in window.webScience) {
+            return;
+        }
+        window.webScience.linkExposureActive = true;
+    }
+    else {
+        window.webScience = {
+            linkExposureActive: true
+        }
+    }
+
     let pageManager = null;
 
     /**
      * How often (in milliseconds) to check the page for new links.
      * @constant {number}
      */
-    const updateInterval = 2000;
+    const updateInterval = 3000;
```

Comment:
We should consider making things like this configurable at runtime, not in this PR but something to think about.

Reply:
Definitely. What I'd like us to support, eventually, is two sets of configurable `linkExposure` options: one that's per-listener (e.g., link and page match patterns) and one that's for all listeners (e.g., timer intervals and self-links).